knowledge-base/getting-to-know-us.md

```diff
@@ -8,15 +8,15 @@ Prezentacje do onboardingu - zostały stworzone issues, osobno dla technicznej i
 
 Po spotkaniu z nową osobą jedna z osób robiących onboarding utrzymuje z nią kontakt – np. przed kolejnym spotkaniem pyta, czy będzie albo po pierwszym spotkaniu pyta, czy ma jakieś pytania etc. 
 
-###### Część I - organizacja
+###### Community
```

Comment:
Mixed english and polish? ;-)

Reply:
That's because `onbarding-checklist.md` is in English and we link to these resources. We should probably revisit the language used here, but that's yet again beyond this pull request's scope.